_data/simple-icons.json

```
@@ -12401,6 +12401,11 @@
             "hex": "FF4154",
             "source": "https://github.com/tannerlinsley/react-table/blob/8c77b4ad97353a0b1f0746be5b919868862a9dcc/docs/src/images/emblem-light.svg"
         },
+        {
+            "title": "React-PDF",
+            "hex": "B4080A",
+            "source": "https://react-pdf.org/images/logo.png"
```

Also, just the website itself will do for the `source` URL.
```suggestion
            "source": "https://react-pdf.org"
```